From 4abd1b934bbf683f4c0938b9835efbac7ca65a31 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=A9=AC=E7=88=B1=E5=9B=BD?= <maaiguo@uniontech.com>
Date: Tue, 10 May 2022 06:31:08 +0000
Subject: add stddef.h for function offsetof

build latest chrome on debian sid  pbuilder env,error below:

net/base/sockaddr_util_posix.cc:24:50: error: expected expression
size_t path_max = address->addr_len - offsetof(struct sockaddr_un, sun_path);

Change-Id: Iabaf1b4502075f524bb47cde1d59cac9190f0599
---
 net/base/sockaddr_util_posix.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/base/sockaddr_util_posix.cc b/net/base/sockaddr_util_posix.cc
index 80146c7bfd851..be5d303410267 100644
--- a/net/base/sockaddr_util_posix.cc
+++ b/net/base/sockaddr_util_posix.cc
@@ -5,6 +5,7 @@
 #include "net/base/sockaddr_util_posix.h"
 
 #include <string.h>
+#include <stddef.h>
 
 #include <sys/socket.h>
 #include <sys/un.h>
-- 
2.34.1

